Lib/test/test_subprocess.py

```diff
@@ -2662,6 +2665,7 @@ def test_select_unbuffered(self):
         finally:
             p.wait()
 
+    @unittest.skipIf(mswindows, "POSIX specific tests")
```

Comment:
This test (ResourceWarning) is still relevant on Windows, no? Just skip the test on _active on Windows.